tasks/extensions/postgis.yml

```diff
@@ -12,3 +12,19 @@
     - "postgresql-{{postgresql_version}}-postgis-scripts"
   notify:
     - restart postgresql
+  when: ansible_os_family == "Debian"
+
+
+- name: PostgreSQL | Extensions | Make sure the postgis extensions are installed | RedHat
+  yum:
+    name: "{{item}}"
+    state: present
+  with_items:
+    - "postgis2_{{postgresql_version_terse}}"
+    - "postgis2_{{postgresql_version_terse}}-client"
+    - "postgis2_{{postgresql_version_terse}}-devel"
```

Comment:
Not necessarily. This could be made conditional on postgresql_ext_install_dev_headers, assuming that if you want the development files for PostgreSQL itself you also want the PostGIS development files. What do you think?